Source/KerbalCombatSystems/MissileGuidance.cs

```diff
@@ -69,12 +69,15 @@ private IEnumerator Launch()
                 seperator.Separate();
 
                 // I will think of something smarter in the future time.
+                // I do not agree with this inclusion, just let badly designed missiles explode - Spartwo
```

Comment:
I'm pretty sure it doesn't work anyway so you can remove it if you like.

Reply:
Based and consequences pilled